tc-report/_site/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>GC Talent | GC Talent</title>
+  <title>GC Digital Talent | GC Digital Talent</title>
   <meta name="description" content="Talent Cloud is an experimental staffing solution created by the Government of Canada. Read our experiment's results report here!">
```

Comment:
Not sure this index.html file is ever served, but can it also use a site variable for the description?

Reply:
Everything in the `_site` directory is autogenerated when you build the site.  So this has changed because of the site variable.

Reply:
It can be served if you manually navigate to the path but it won't be served by the server as a "real" index.  It can't actually be localized (I think) because it's not under /en or /fr.  I've gone ahead and removed it totally.
[Remove tc-report index and 404 pages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4468/commits/423685bf337d2564c36249d70518b1f87b957617)